bundles/org.eclipse.ui.editors/src/org/eclipse/ui/texteditor/AbstractDecoratedTextEditorPreferenceConstants.java

```diff
@@ -723,6 +747,8 @@ private AbstractDecoratedTextEditorPreferenceConstants() {
 	* @param store the preference store to be initialized
 	*/
 	public static void initializeDefaultValues(IPreferenceStore store) {
+		store.setDefault(AbstractDecoratedTextEditorPreferenceConstants.EDITOR_USE_FIND_REPLACE_OVERLAY, false);
```

Comment:
Sorry for the back forth: since we now have to target 2024-09 M1, we should directly enable the new overlay by default (whereas we disabled it by default when targeting the 2024-06 release).
```suggestion
		store.setDefault(AbstractDecoratedTextEditorPreferenceConstants.EDITOR_USE_FIND_REPLACE_OVERLAY, true);
```

Reply:
https://github.com/eclipse-platform/eclipse.platform.ui/commit/d2f3b6eb5a5cbd69011cf24ea8b05e9722212ff5